package.json

```diff
@@ -29,6 +29,7 @@
     "lint-staged": "^10.5.4",
     "parcel-bundler": "^1.12.5",
     "prettier": "^2.2.1",
+    "react": "^17.0.2",
```

Comment:
I think we need `react` at run-time don't we? It probably shouldn't be a _dev_ dependency.

Reply:
It's now been switched, no idea how it ended up there...